tests/array.rs

```diff
@@ -2466,74 +2463,48 @@ mod array_cow_tests {
 
 #[test]
 fn test_remove_index() {
-    let mut a = arr2(&[[1, 2, 3],
-                       [4, 5, 6],
-                       [7, 8, 9],
-                       [10,11,12]]);
+    let mut a = arr2(&[[1, 2, 3], [4, 5, 6], [7, 8, 9], [10, 11, 12]]);
     a.remove_index(Axis(0), 1);
     a.remove_index(Axis(1), 2);
     assert_eq!(a.shape(), &[3, 2]);
-    assert_eq!(a,
-        array![[1, 2],
```

Comment:
Sorry to nitpick, but when the code is arranged to be easy to read (multiline array), it should not be reformatted.

Reply:
I have my editor set to automatically run rustfmt. I'll revert the format changes to this file.